protocol/communities/community.go

```diff
@@ -39,6 +39,7 @@ type Config struct {
 	RequestsToJoin                []*RequestToJoin
 	MemberIdentity                *ecdsa.PublicKey
 	SyncedAt                      uint64
+	AdminsEvents                  []CommunityAdminEvent
```

Comment:
Small detail, but technically a typo
```suggestion
	AdminEvents                  []CommunityAdminEvent
```